gradle-plugin/gradlew.properties

```
@@ -0,0 +1 @@
+org.gradle.jvmargs="-Xms512m -Xmx2048m"
```

```suggestion
org.gradle.jvmargs="-Xms512m"
```

An approach we've used in other builds is to only set the minimum/start memory limits. That indicates what's at least necessary to run (or start faster). Setting the maximum unnecessarily limits the build for users with powerful machines.

---

`-Xmx` is useful to force GC, sometimes it may prevent OOM in Groovy \ Gradle tests with multiple forked JVMs. But I can remove `-Xmx`